bin/build-cored-release

```diff
@@ -48,7 +48,7 @@ DATE=${DATE:-`date +%s`} # date can be set via envvar
 ldflags="-X main.buildTag=$releaseRef -X main.buildCommit=$commit -X main.buildDate=$DATE"
 
 go build\
-  -tags 'plain_http'\
+  -tags 'plain_http loopback_auth'\
```

Comment:
Discussed out of band: the product folks are using this for developer edition builds. A search of the codebase shows it only being used for this purpose as well. Should I add a comment about the flags here?